### PR TITLE
Fix 496: Decoder behaviour for redundant copies

### PR DESCRIPTION
**Pull request by felicialim**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/501.html" title="Last updated on Jun 27, 2023, 11:57 PM UTC (e32896a)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/501/6ac4b27...e32896a.html" title="Last updated on Jun 27, 2023, 11:57 PM UTC (e32896a)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -1730,9 +1732,9 @@ This section details the order in which the OBUs are sequenced in a standalone I
 
 An <dfn noexport>IA Sequence</dfn> is composed of a series of OBUs in the sequence of a set of [=Descriptors=] followed by their associated [=IA Data=]s. 
 
-NOTE: In typically case, the first [=Descriptors=] of the [=IA Sequence=] are all non-redundant (i.e. [=obu_redundant_copy=] = 0).
+The first [=Descriptors=] of the [=IA Sequence=] SHALL be marked as non-redundant (i.e. [=obu_redundant_copy=] = 0).
```
Comment (sunghee-hwang) [inline index.bs:1735]: The original `IA Sequence` may be edited to an `IA Sequence` starting with the second redundant Descriptors.

```suggestion
NOTE: In typically case, the first [=Descriptors=] of the [=IA Sequence=] are all non-redundant (i.e. [=obu_redundant_copy=] = 0).
```

**Review by sunghee-hwang — COMMENTED** [review]
The original `IA Sequence` may be edited to an `IA Sequence` starting with the second redundant Descriptors.

File: index.bs (hunk 1 above)
Comment (felicialim) [reply index.bs]: Done

File: index.bs [hunk 2]
```diff
@@ -457,6 +457,8 @@ It SHALL always be set to 0 for the following [=obu_type=] values:
 - OBU_IA_Audio_Frame
 - OBU_IA_Audio_Frame_ID0 to OBU_IA_Audio_Frame_ID21
 
+If a decoder encounters an OBU with [=obu_redundant_copy=] = 1, and it has also received the previous non-redundant OBU, it SHALL ignore the redundant OBU. If the decoder has not received the previous non-redundant OBU, it SHALL treat the redundant copy as a non-redundant OBU and process the OBU accordingly.
```
Comment (sunghee-hwang) [inline index.bs:460]:
```suggestion
For OBUs with a given [=obu_type=] in an [=IA Sequence=], a decoder SHALL treat the first OBU, which the decoder encounters, as a non-redundant OBU regardless of [=obu_redundant_copy=] = 0 or 1 and process the OBU accordingly. After that, if the decoder encounters the OBU with [=obu_redundant_copy=] = 1, it is RECOMMENDED for the decoder to ignore the redundant OBU.
```

**Review by sunghee-hwang — APPROVED** [review]

**Review by cconcolato — APPROVED** [review]

**Pull request merged**